packages/builder/src/services/pinata.ts

```diff
@@ -17,7 +17,7 @@ export default class PinataClient {
   constructor() {
     if (JWT === undefined || GATEWAY === undefined) {
       throw new Error(
-        "remember to set the REACT_APP_SUBGRAPH_MAINNET_API_KEY and REACT_APP_PINATA_GATEWAY env vars"
+        "remember to set the REACT_APP_SUBGRAPH_URL_MAINNET and REACT_APP_PINATA_GATEWAY env vars"
```

Comment:
The message seems to be wrong anyways no? it's checking for the gateway and the JWT

```suggestion
        "remember to set the REACT_APP_PINATA_JWT and REACT_APP_PINATA_GATEWAY env vars"
```